src/view/com/auth/SplashScreen.web.tsx

```diff
@@ -98,22 +99,23 @@ export const SplashScreen = ({
 
 function Footer({styles}: {styles: ReturnType<typeof useStyles>}) {
   const pal = usePalette('default')
+  const {_} = useLingui()
 
   return (
     <View style={[styles.footer, pal.view, pal.border]}>
       <TextLink
         href="https://blueskyweb.xyz"
-        text="Business"
+        text={_(msg`Business`)}
```

Comment:
Just because these are business links we probably don't want this text localized

Reply:
But in `HomeLoggedOutCTA.tsx`, it was already localized.